sql/updates/world/2016_01_12_12_world335.sql

```diff
@@ -0,0 +1,5 @@
+-- Insert missing gossip_menu_option 21213 for Sorcerer Ashcrombe and 21214 for Deathstalker Adamant:
+DELETE FROM `gossip_menu_option` WHERE `menu_id` IN (21213,21214) AND `OptionBroadcastTextID`= 2802;
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`) VALUES
+(21213,0,0,'Please unlock the courtyard door.',2802,1,1,0,0,0,0,'',0),
+(21214,0,0,'Please unlock the courtyard door.',2802,1,1,0,0,0,0,'',0);
```

Comment:
"Thanks, I'll follow you to the door." is a made-up text and not used by Blizzard as far as I could find. The broadcast_text ID 2802 is correct for these 2 NPCs. Please also note that there are 2 different `gossip_menu_option`.`menu_id`s, 1 for each NPC (21213 for alliance, 21214 for horde). You would probably say that it doesn't matter, but the 2x `gossip_menu`.`entry` values also need to link to 2 different `text_id`s, a.k.a. `npc_text`.`ID` 798 and 799. Those already existed in TDB 335.60:
```sql
SELECT * FROM `gossip_menu` WHERE `entry` IN (21213,21214);
```
```
 entry  text_id
------  ---------
 21213      798
 21214      799
```
The way I see it, I have followed the rules to the letter. If you can find sniffs contradicting this, I would think that the `broadcast_text` and `gossip_menu`  needs to be revised for these 2 NPCs.